### PR TITLE
[mqttembeddedbroker] add deprecation note

### LINKED ISSUE #7442: [mqtt][moquette] Formally deprecate embedded moquette?
It is generally accepted in the user community that the Embedded MQTT broker has been abandoned by its original developers, and as a few issues have emerged this year, there is a general recommendation not to use it.
https://community.openhab.org/t/mqtt-embedded-broker-advantages/88441/7

The official OH docs remain enthusiastic about it, and new users continue to install it.
https://www.openhab.org/addons/integrations/mqttembeddedbroker/
https://community.openhab.org/t/mqtt-broker-version/97204/2

It's above my pay grade to go adding warning messages to the docs about deprecating it, this I feel is a policy decision for some developer.

### PR DESCRIPTION
**Pull request by J-N-K**
Closes #7442 

Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by bodiroga** [comment]
I suppose that removing the addon is not possible because people may still be using it (although it should be removed for OH3), but what about adding "deprecated" to its name? I fear that people will continue installing it, because not everyone reads the documentation and the added warning will be ignored :cry:

**Comment by J-N-K** [comment]
I agree that documentation is not read, but I think this is the only way until OH3 is releaSed.

File: bundles/org.openhab.io.mqttembeddedbroker/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # MQTT Broker Moquette
 
+**Attention:** Moquette is no longer maintained upstream and this addon is therefore considered deprecated in openHAB.
```
Comment (kaikreuzer) [inline bundles/org.openhab.io.mqttembeddedbroker/README.md:3]:
```suggestion
**Attention:** Moquette is no longer maintained upstream and this add-on is therefore considered deprecated in openHAB. 
```

File: bundles/org.openhab.io.mqttembeddedbroker/README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # MQTT Broker Moquette
 
+**Attention:** Moquette is no longer maintained upstream and this addon is therefore considered deprecated in openHAB. 
+It is recommended to switch to an external broker like mosquitto.
+The Moquette addon will be removed in a future release of openHAB.
```
Comment (kaikreuzer) [inline bundles/org.openhab.io.mqttembeddedbroker/README.md:5]:
```suggestion
The Moquette add-on will be removed in a future release of openHAB.
```

**Pull request merged**